crates/wallet/tests/wallet.rs

```diff
@@ -4113,7 +4109,7 @@ fn test_keychains_with_overlapping_spks() {
         .address;
     let chain_position = ChainPosition::Confirmed(ConfirmationBlockTime {
         block_id: BlockId {
-            height: 8000,
+            height: 2000,
```

Comment:
I might've not noticed it properly, but why was this change necessary?

Reply:
I think the height of the anchor block was inadvertently changed in #1643 and this changes it back to the way it was as of 1.0.0-beta.5

Reply:
@ValuedMammal Woops, thanks for picking up on that!